README.md

```diff
@@ -90,15 +92,6 @@ Versions are auto incremented based on commit messages and SemVer (Major.Minor.P
 - `patch:` or `fix:` example `fix: url bug` 
   - this will increment the PATCH version eg 1.0.0 to 1.0.1. This will also happen by default.
 
-#### Build process
-
-1. Pushes to any branch will start the build process
-2. `.circleci/circle_trigger.sh` will check for any changes in our packages, and if builds have failed previously
-3. `circle_trigger.sh` will trigger a workflow via the API. It will pass the parameters model, runner, designer (bool) to the workflow.
```

Comment:
Hi! Should this be as follows? And should we use 'bool' or 'boolean' here? 

3. `circle_trigger.sh` will trigger a workflow via the API. It will pass the parameters `model`, `runner`, `designer` (bool) to the workflow.